Polish translation

michalsapka opened the pull request:
Polish translation for 1.3.0

Barthalion commented on version/1/3/0/pl/code_of_conduct.md:
```diff
@@ -0,0 +1,24 @@
+
+My, współtwórcy, działając dla dobra oraz otwartości naszej społeczności, ślubujemy poszanowanie dla wszystkich osób które wnoszą wartość do naszej pracy. Wartość tą wnoszą między innymi poprzez: zgłaszanie problemów, propozycji zmian, aktualizację dokumentacji, czy zgłaszanie poprawek poprzez "pull request".
+
+Jesteśmy zdeterminowani, by współpraca nad tym projektem była dla wszystkich wolna od zastraszania niezależnie od: doświadczenia; płci; identyfikacji, ekspresji i orientacji seksualnej; kalectwa; wyglądu; rozmiaru ciała; rasy; przynależności etnicznej; wieku; wyznawanej religii ani narodowości.
```
I think colons should be used instead of semicolons.

michalsapka replied:
(in polish)

"poszczególne punkty wyliczenia zamyka się jakimś znakiem interpunkcyjnym. Może to być przecinek (gdy punkty są krótkie), _średnik (gdy są dłuższe i same zawierają przecinki) albo kropka (gdy punkty mają charakter pełnych zdań)_."
more: http://sjp.pwn.pl/poradnia/haslo/interpunkcja-w-wyliczeniach;1536.html

Barthalion commented:
LGTM

Witos commented:
Ponglish mistakes, like 'zdeterminowani'

michalsapka commented:
PWN says "zterminowany" is correct - http://sjp.pwn.pl/sjp/zdeterminowany;2545230.html

CoralineAda commented on version/1/3/0/pl/code_of_conduct.md:
```diff
@@ -0,0 +1,24 @@
+
+Przypadki obraźliwego, zastraszającego lub w inny sposób nieakceptowalnego zachowania mogą być zgłaszane do opiekuna tego projektu drogą elektroniczną pod adresem [TWÓJ ADRES EMAIL]. Wszystkie zażalenia i uwagi zostaną rozpatrzone i będą podstawą do analizy. W przypadku potwierdzenia się zarzutów zostaną podjęte odpowiednie do przewinienia środki zaradcze. Opiekunowie tego projektu są zobligowani ...
+
+Niniejszy Kodeks postępowania jest zgodny z [Contributor Covenant](http://contributor-covenant.org) w wersji 1.3.0. Oryginalny tekst można znaleźć pod adresem: [http://contributor-covenant.org/version/1/3/0/](http://contributor-covenant.org/version/1/3/0/)
```
/1/3/0/pl/

Witos replied:
ślubujemy poszanowanie dla wszystkich osób które wnoszą wartość do naszej pracy. Wartość tą wnoszą między innymi poprzez: zgłaszanie problemów, propozycji zmian, aktualizację dokumentacji, czy zgłaszanie poprawek poprzez "pull request". -> ślubujemy poszanowanie dla wszystkich osób które wnoszą wkład do naszej pracy poprzez: zgłaszanie problemów, propozycji zmian, aktualizację dokumentacji, czy zgłaszanie poprawek poprzez "pull request"

Translation of the next paragraph sounds a bit akward in polish, I suggest: 
"We are committed to making participation in this project a
harassment-free experience for everyone, regardless of level of
experience, gender, gender identity and expression, sexual orientation,
disability, personal appearance, body size, race, ethnicity, age,
religion, or nationality." -> "Pragniemy, aby udział w projekcie był pozbawiony szykanowanowania ze względu na doświadczenie, płeć, orientację seksualną, stopień niepełnosprawności, wygląd, budowę ciała, kolor skóry, pochodzenie, czy wiek.

michalsapka replied:
Może "(...)*bez* względu na(...)"?

Witos replied:
IMHO "pozbawiony szykanowania bez względu na (...)" doesn't sound right, because "bez względu na" shouldn't have a denial in the first part.

Witos replied:
There should be a comma before "bądź"

CoralineAda commented:
After you make the change in the link and create the html and txt versions I'll be ready to merge! Thanks so much!

michalsapka commented:
I've update the text
- changed both sentects (as per @Witos proposition). I've added sexual "identity" because it is not the same thing as "preference". There is also "expression" in the original text, but I've never met this in any polish media.
- I'm not sure about the coma before "bądź" - http://www.rjp.pan.pl/index.php?option=com_content&id=972:przecinek-przed-qbdq&Itemid=58

Witos commented:
@michalsapka sorry, you're right about "bądź"

michalsapka commented:
No problem :)

If the text is OK with you, I'll add the .html and .txt version in the evening and it will be ready for merge.

CoralineAda commented:
Go for it!

Witos commented:
@michalsapka lgtm

michalsapka commented:
@CoralineAda ready.
